src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,45 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
+    private Long id;
+    private String name;
+    private String country;
+
+    public Manufacturer() {
+    }
+
+    public Manufacturer(String name, String country) {
```

Comment:
You can read about this 3 annotation for your constructor to create it of course if you dont use custom logic for your constructor, link: https://projectlombok.org/features/constructor